src/services/certificationService.ts

```
@@ -1,33 +1,106 @@
-      method: 'GET',
-      headers: {
-        'Content-Type': 'application/json',
-        ...(token && { 'Authorization': `Bearer ${token}` })
+import { CertificateDetail } from "@/components/ui/certificate";
+
+export class CertificationService {
+  private static readonly BASE_URL = "http://localhost:8080/api";
+
+  static async getCertificates(userId: string, token: string) {
+    try {
+      const res = await fetch(`${this.BASE_URL}/api/certificates/${userId}`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)